pyne/alara.py

```diff
@@ -128,7 +128,11 @@ def photon_source_to_hdf5(filename, chunkshape=(10000,)):
     Parameters
     ----------
     filename : str
-        The path to the file
+        The path to the file for read
+    nucs : str
+        Nuclides need to write into h5 file. For example:
+            - 'all': default value. Write the information of all nuclides to h5.
+            - 'total': used for r2s. Only write TOTAL value to h5.
```

Comment:
Should you check somewhere that it’s one of those options?  Right now if it’s not, it will proceed without error or warning and do nothing

Reply:
This check has been added in 28c5717 and 859090c.